### PR TITLE
[TYPO] fix typo/format in quicktour.md

### PR DESCRIPTION
**Pull request by lishukan**
# What does this PR do?

Fixes some typo/format error:
image

image

It's  due to a lack of blank lines before code markup.

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@sgugger, @stevhliu and @MKhalusova

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/fr/quicktour.md [hunk 1]
```diff
@@ -23,18 +23,20 @@ Soyez opérationnel avec 🤗 Transformers ! Que vous soyez un développeur ou u
 Avant de commencer, assurez-vous que vous avez installé toutes les bibliothèques nécessaires :
 
 ```bash
-!pip install transformers datasets
+pip install transformers datasets
```
Comment (sgugger) [inline docs/source/fr/quicktour.md:26]: No the exclamation mark needs to stay for the notebook conversion of this doc page.

File: docs/source/ko/quicktour.md [hunk 2]
```diff
@@ -23,18 +23,20 @@ rendered properly in your Markdown viewer.
 시작하기 전에 필요한 라이브러리가 모두 설치되어 있는지 확인하세요:
 
 ```bash
-!pip install transformers datasets
+pip install transformers datasets
```
Comment (sgugger) [inline docs/source/ko/quicktour.md:26]: Same here.

File: docs/source/en/quicktour.md [hunk 3]
```diff
@@ -23,18 +23,20 @@ Get up and running with 🤗 Transformers! Whether you're a developer or an ever
 Before you begin, make sure you have all the necessary libraries installed:
 
 ```bash
-!pip install transformers datasets
+pip install transformers datasets
```
Comment (sgugger) [inline docs/source/en/quicktour.md:26]: The exclamation mark needs to stay for the conversion of this doc page into notebook.

File: docs/source/zh/quicktour.md [hunk 4]
```diff
@@ -23,18 +23,20 @@ rendered properly in your Markdown viewer.
 在开始之前, 确保你已经安装了所有必要的库:
 
 ```bash
-!pip install transformers datasets
```
Comment (sgugger) [inline docs/source/zh/quicktour.md:26]: Same here.

**Review by sgugger — COMMENTED** [review]
Thanks for the fixes. The exclamation mark in bash comments is intended since those doc pages are auto-converted into notebooks, please revert those changes.

**Review by sgugger — APPROVED** [review]
Thanks for iterating!

File: docs/source/ko/quicktour.md (hunk 2 above)
Comment (lishukan) [reply docs/source/ko/quicktour.md]: revert done

File: docs/source/en/quicktour.md (hunk 3 above)
Comment (lishukan) [reply docs/source/en/quicktour.md]: done

**Comment by lishukan** [comment]
@sgugger    Hi, Dear sgugger   , I met some problem on this PR.
It failed on the test [ check_circleci_user  ]. what should i do can solve it ?
I've spend lots of time to search document/question about this ，and the 
question link: [ https://support.circleci.com/hc/en-us/articles/360008097173-Troubleshooting-why-pull-requests-are-not-triggering-jobs-on-my-organization- ](https://support.circleci.com/hc/en-us/articles/360008097173-Troubleshooting-why-pull-requests-are-not-triggering-jobs-on-my-organization-) in test log  was invalid. 
Only i can do is just looking help from you,sorry about  waste your precious time

 
image

**Review by sgugger — APPROVED** [review]
I've pushed your branch to the main fork of Transformers, which re-triggers the tests. Normally there shouldn't be any problem since you are only touching doc files, but let's double-check!

**Comment by lishukan** [comment]
> I've pushed your branch to the main fork of Transformers, which re-triggers the tests. Normally there shouldn't be any problem since you are only touching doc files, but let's double-check!

Tanks a lot

**Pull request merged**